src/components/Map/hooks/useHighlight.js

```diff
@@ -0,0 +1,41 @@
+import { useRef } from 'react';
+import 'leaflet.markercluster';
+
+export const HIGHLIGHT_CLASS = 'active-element';
+
+const useHighlight = () => {
+  const activeMarker = useRef(null);
+  const activePolygon = useRef(null);
+
+  const clearSelection = () => {
```

Comment:
Is it intended that `highlightMarker` clears both highlighted markers and polygons instead of just markers?